src/main.cpp

```diff
@@ -21,8 +22,8 @@ int main(int argc, char *argv[])
     /* Set permissions for the new files */
     QFile mlbInfo("MLBInformation.csv");
     QFile distance("DistanceBetweenStadiums.csv");
-    mlbInfo.setPermissions(QFileDevice::Permissions(0777));
-    distance.setPermissions(QFileDevice::Permissions(0777));
+    mlbInfo.setPermissions(QFileDevice::Permissions(07777));
```

Comment:
ew, these are still not executables, why do they have +x, uid, gid, and sticky bit?